src/tooltip.js

```diff
@@ -98,7 +102,12 @@ nv.models.tooltip = function() {
 
         trowEnter.append("td")
             .classed("value",true)
-            .html(function(p, i) { return valueFormatter(p.value, i, p) });
+            .html(function(p, i) {
+                if (enableComplexValueFormatter) { // Mirror Chart
+                    return valueFormatter(p, i);
+                }
+                return valueFormatter(p.value, i, p);
```

Comment:
Agree, as long as charts work as it's defined by `nvd3` documentation, we shouldn't worry about their internal code organization.